gateway-js/src/index.ts

```diff
@@ -772,7 +771,12 @@ export class ApolloGateway implements GraphQLService {
     // 2) non-empty query plan and shouldShowQueryPlan === true
     const serializedQueryPlan =
       queryPlan.node && (this.config.debug || shouldShowQueryPlan)
-        ? serializeQueryPlan(queryPlan)
+        // FIXME: I disabled printing the query plan because this lead to a
```

Comment:
I do think this is a feature that people like. We can maybe duplicate the code with comments for the meantime?

Worth noting that workbench is almost certainly using the `serializeQueryPlan` fn - we probably want the gateway package (or some other published package) to keep this fn publicly exposed.